### ISSUE TITLE
chrono::Duration (chrono::TimeDelta) not supported even with chrono feature

### ISSUE THREAD

**Issue by doivosevic** [GREsau/schemars#357]
I'm on chrono 0.4.38 and schemars 0.8.21.
I have chrono feature enabled for schemars.
The compiler is complaining that JsonSchema is not implemented for `TimeDelta`. Is this expected?

```
the trait bound `TimeDelta: JsonSchema` is not satisfied
the following other types implement trait `JsonSchema`:
  &'a T
  &'a mut T
  ()
  (T0, T1)
  (T0, T1, T2)
  (T0, T1, T2, T3)
  (T0, T1, T2, T3, T4)
  (T0, T1, T2, T3, T4, T5)
and 278 others
required for `(NaiveTime, TimeDelta)` to implement `JsonSchema`
1 redundant requirement hidden
required for `std::vec::Vec<(NaiveTime, TimeDelta)>` to implement `JsonSchema`rustc[Click for full compiler diagnostic](rust-analyzer-diagnostics-view:/diagnostic message [0]?0#file:///git/qwe.rs)
_private.rs(126, 43): required by a bound in `insert_object_property
```
